symplyphysics/laws/thermodynamics/relative_humidity.py

```diff
@@ -0,0 +1,46 @@
+from sympy import (Eq, solve, S)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless,
+    convert_to
+)
+
+# Description
+## Relative humidity is the ratio of water vapor pressure p to saturated vapor pressure p0 at a given temperature.
+
+## Law is: ph = (p / p_sat) * 100, where
+## ph - relative humidity,
+## p - water vapor pressure,
+## p_sat - saturated vapor pressure.
+
+# Conditions:
+## - the process is isobaric-isothermal.
+
+relative_humidity = Symbol("relative_humidity", dimensionless)
+
+water_vapor_pressure = Symbol("water_vapor_pressure", units.pressure)
+saturated_vapor_pressure = Symbol("saturated_vapor_pressure", units.pressure)
+
+law = Eq(relative_humidity, (water_vapor_pressure / saturated_vapor_pressure) * 100)
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(water_vapor_pressure_=water_vapor_pressure,
+    saturated_vapor_pressure_=saturated_vapor_pressure)
+@validate_output(relative_humidity)
+def calculate_relative_humidity(water_vapor_pressure_: Quantity,
+    saturated_vapor_pressure_: Quantity) -> float:
```

Comment:
You can return Quantity to let users convert the result to percents automatically.